pwndbg/commands/__init__.py

```diff
@@ -44,13 +53,14 @@ class CommandCategory(str, Enum):
 
 def list_current_commands():
     current_pagination = gdb.execute("show pagination", to_string=True)
-    current_pagination = current_pagination.split()[-1].rstrip(
-        "."
-    )  # Take last word and skip period
+    if current_pagination is not None:
```

Comment:
Redundant if; pagination should always exist there in our GDB session afaik

Reply:
fixed!